src/startupscriptgenerator/src/common/parseUserCommands.go

```diff
@@ -0,0 +1,61 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+package common
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func ParseUserRunCommand(sourcePath string) string {
+
+	appsvcFile, err := os.Open(sourcePath)
+	if err != nil {
+		fmt.Println(err)
+		return ""
+	}
+	defer appsvcFile.Close()
+
+	const runHeading string = "run:" // format of run command- "run: gunicorn myapp.app --workers 5"
+	runCommand := ""
+	var isRunCommandFound bool
+	scanner := bufio.NewScanner(appsvcFile)
+
+	for scanner.Scan() {
+		//fmt.Println(scanner.Text())
+		indexOfRunHeading := strings.Index(scanner.Text(), runHeading)
+		isRunHeadingExists := indexOfRunHeading > 0
```

Comment:
won't this indicate that the run heading doesn't exist if it is in the very first position e.g. `run: a temp command`? It may be better to check if `indexOfRunHeading` is equal to `-1`, indicating that it was not found anywhere in the string?